base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -345,7 +349,20 @@ func (vgm *virtualGroupManager) refreshMetaByChain() {
 func (vgm *virtualGroupManager) PickVirtualGroupFamily() (*vgmgr.VirtualGroupFamilyMeta, error) {
 	vgm.mutex.RLock()
 	defer vgm.mutex.RUnlock()
-	return vgm.vgfManager.pickVirtualGroupFamily()
+	vgfIDS := make([]uint32, len(vgm.vgfManager.vgfIDToVgf))
+	if len(vgfIDS) == 0 {
+		return nil, ErrFailedPickVGF
+	}
+	i := 0
+	for k := range vgm.vgfManager.vgfIDToVgf {
+		vgfIDS[i] = k
+		i++
+	}
+	availableVgfIDs, err := vgm.chainClient.AvailableGlobalVirtualGroupFamilies(context.Background(), vgfIDS)
+	if err != nil {
+		return nil, err
+	}
+	return vgm.vgfManager.pickVirtualGroupFamily(availableVgfIDs)
```

Comment:
The `pickGlobalVirtualGroupForBucketMigrate`  API also needs attention, because it may also choose family.